benchmarks/BenchmarkServer/README.md

```diff
@@ -0,0 +1,8 @@
+## Usage
```

Comment:
Could you also add a **Purpose** for the app to the readme for what it is used for so someone who doesn't know, i.e. me, can quickly find out.